### PR TITLE
fix issue #4

### PR DESCRIPTION
**Pull request by pavelpikta**
fix issue #4

### CODE REVIEW

File: tasks/oracle-db-11-12.yml [hunk 1]
```diff
@@ -92,3 +90,45 @@
   tags:
     - skip_ansible_lint
   become: True
+
+- name: "Stop Oracle database"
+  shell: >
+    {{ oracledb.oracle_home }}/bin/dbshut {{ oracledb.oracle_home }}
+  tags:
+    - skip_ansible_lint
```
Comment (anastacia-maletskaya-epam) [inline tasks/oracle-db-11-12.yml]: we have an issue in ansible-lint regarding "skip_ansible_lint" https://github.com/lean-delivery/ansible-lint-rules/issues/14 
After implementing CI in this role will fail

File: tasks/oracle-db-11-12.yml [hunk 2]
```diff
@@ -92,3 +90,45 @@
...
+  become: True
+
+- name: "Setup database port"
```
Comment (anastacia-maletskaya-epam) [inline tasks/oracle-db-11-12.yml]: this block should not be executed if we don't change default port?

File: tasks/oracle-db-11-12.yml (hunk 1 above)
Comment (anastacia-maletskaya-epam) [reply tasks/oracle-db-11-12.yml]: All skip_ansible_lint tags should be removed

File: defaults/main.yml [hunk 3]
```diff
@@ -48,6 +48,7 @@ pdbadminpassword: "PdbAdminPassword2017SecretToken"
 
 db_user: "oracle"
 db_tablespace: "oracle"
+db_port: 1521
 
 ansible_ssh_pipelining: True
```
Comment (tgadiev) [inline defaults/main.yml]: Why hardcoding ansible_ssh settings?

File: tasks/system/prepare.yml [hunk 4]
```diff
@@ -56,9 +56,12 @@
...
 - name: "Install requirements"
   package:
-    name: "{{ item }}"
+    name: "{{ oracledb.install_requirements }}"
     state: present
-  loop: "{{ oracledb.install_requirements }}"
+  register: status
+  retries: 10
+  delay: 2
+  until: status is success
```
Comment (tgadiev) [inline tasks/system/prepare.yml:62]: Comment for lines 89 & 106:
Is tag skip_ansible_lint really necessary here? I don't see any reasons for that case.

File: tasks/system/prepare.yml [hunk 5]
```diff
@@ -56,9 +56,12 @@
...
   become: True
 
 - name: "Fetch artifact with {{ transport }} transport"
```
Comment (tgadiev) [inline tasks/system/prepare.yml:65]: Comment for lines 87:
Conditional will be better look this way:
```
when: not database.stat.exists
```

File: tasks/system/swap.yml [hunk 6]
```diff
@@ -18,6 +18,8 @@
       command: "dd if=/dev/zero of={{ oracledb.swapfile_path }} \
```
Comment (tgadiev) [inline tasks/system/swap.yml]: Task will be better looking in folded scalar style:
```yaml
    - name: "Create swapfile"
      command: >-
        dd if=/dev/zero of={{ oracledb.swapfile_path }}
        bs={{ oracledb.swapfile_bs_size_mb }}M
        count={{ oracledb.swapfile_count }}
      args:
        creates: "{{ oracledb.swapfile_path }}"
      when: not swapfile.stat.exists
```

File: tasks/system/swap.yml [hunk 7]
```diff
@@ -27,9 +29,13 @@
 
     - name: "Make swap"
       command: "/sbin/mkswap {{ oracledb.swapfile_path }}"
+      tags:
+        - skip_ansible_lint
```
Comment (tgadiev) [inline tasks/system/swap.yml]: Why is it here?

File: tasks/system/swap.yml [hunk 8]
```diff
@@ -27,9 +29,13 @@
...
 
     - name: "Swap on"
       command: "/sbin/swapon {{ oracledb.swapfile_path }}"
+      tags:
+        - skip_ansible_lint
```
Comment (tgadiev) [inline tasks/system/swap.yml]: Is it necessary?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: tasks/oracle-db-11-12.yml (hunk 2 above)
Comment (vnovichek) [reply tasks/oracle-db-11-12.yml]: second, doesn't make sense to setup the port if it hasn't been changed in the first place, fixed

File: defaults/main.yml (hunk 3 above)
Comment (vnovichek) [reply defaults/main.yml]: no clue, it seems someone was trying to tune molecule ages ago, got rid of it

File: tasks/system/prepare.yml (hunk 4 above)
Comment (vnovichek) [reply tasks/system/prepare.yml:62]: me neither, thrown away

File: tasks/system/prepare.yml (hunk 5 above)
Comment (vnovichek) [reply tasks/system/prepare.yml:65]: right, fixed

File: tasks/system/swap.yml (hunk 6 above)
Comment (vnovichek) [reply tasks/system/swap.yml]: agree, fixed

File: tasks/system/swap.yml (hunk 7 above)
Comment (vnovichek) [reply tasks/system/swap.yml]: removed

File: tasks/system/swap.yml (hunk 8 above)
Comment (vnovichek) [reply tasks/system/swap.yml]: not at the moment, but as soon as ansible-lint is fixed, it'll shout:

[E511] Shell/command module must contain creates or removes
/home/vagrant/1/ldi/tasks/system/swap.yml:31
Task/Handler: Swap on

**Review by tgadiev — DISMISSED** [review]

File: tasks/oracle-db-11-12.yml (hunk 1 above)
Comment (vnovichek) [reply tasks/oracle-db-11-12.yml]: all skip_ansible_lint tags have been removed

File: tasks/system/swap.yml (hunk 8 above)
Comment (vnovichek) [reply tasks/system/swap.yml]: fixed

**Review by tgadiev — APPROVED** [review]

**Pull request merged**